AdminOpsGuide/dbrepl.rst

```diff
@@ -1191,141 +1193,312 @@ Here is the code:
+   Wed Jul 18 12:46:57 2018 : Initiating CHECKHEALTH operation on source server pid [18529] for secondary instance name [dummy]
+   PID 18529 Source server is alive in PASSIVE mode
+   Wed Jul 18 12:46:57 2018 : Initiating SHOWBACKLOG operation on source server pid [18529] for secondary instance [dummy]
+   0 : backlog number of transactions written to journal pool and yet to be sent by the source server
+   0 : sequence number of last transaction written to journal pool
+   0 : sequence number of last transaction sent by source server
+   WARNING - Source Server is in passive mode, transactions are not being replicated
+   -----------------------------------------------------------------
+   Receiver Server B:
+   -----------------------------------------------------------------
+   PID 18531 Receiver server is alive
+   PID 18532 Update process is alive
+   0 : number of backlog transactions received by receiver server and yet to be processed by update process
+   0 : sequence number of last transaction received from Source Server and written to receive pool
+   0 : sequence number of last transaction processed by update process
 
 The shutdown sequence is as follows:
 
+On B:
+
 .. parsed-literal::
-   source ./ydbenv B r1.20_x86_64
-   ./replicating_stop
-   source ./env A r1.20_x86_64
-   ./originating_stop
+   $./replicating_stop
+   Wed Jul 18 13:02:30 2018 : Waiting for 2 second(s) before forcing shutdown
+   Wed Jul 18 13:02:32 2018 : Initiating shut down
+   Wed Jul 18 13:02:33 2018 : Receive pool shared memory removed
+   Wed Jul 18 13:02:33 2018 : Receive pool semaphore removed
+   Wed Jul 18 13:02:33 2018 : Waiting for 2 second(s) before forcing shutdown
+   Wed Jul 18 13:02:35 2018 : Initiating SHUTDOWN operation on source server pid [18529] for secondary instance [dummy]
+   Wed Jul 18 13:02:35 2018 : Waiting for upto [120] seconds for the source server to shutdown
+   Wed Jul 18 13:02:36 2018 : Journal pool shared memory removed
+   Wed Jul 18 13:02:36 2018 : Journal pool semaphore removed
+
+On A:
+
+.. parsed-literal::
+   $./originating_stop
+   Wed Jul 18 13:02:59 2018 : Waiting for 2 second(s) before forcing shutdown
+   Wed Jul 18 13:03:01 2018 : Initiating SHUTDOWN operation on source server pid [18498] for secondary instance [B]
+   Wed Jul 18 13:03:01 2018 : Waiting for upto [120] seconds for the source server to shutdown
+   Wed Jul 18 13:03:02 2018 : Journal pool shared memory removed
+   Wed Jul 18 13:03:02 2018 : Journal pool semaphore removed
+   %YDB-I-MUFILRNDWNSUC, File /home/A/yottadb.dat successfully rundown
 
 +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 Setting up an A -> B -> C Replication Configuration with Empty Databases
 +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-On A:
+**On A**:
+
+* Source the environment file.
+
+.. parsed-literal::
+   source ./ydbenv A r122
 
-* Turn on replication.
 * Create the replication instance file.
```

Comment:
The below step creates the database and replication instance file. Not just instance file. Copy the text from the previous section (line 1263). Can you incorporate the suggestions till now across all the below sections. I will review them after that. Thanks.